src/pages/index.tsx

```diff
@@ -9,54 +9,110 @@ import {
+                                Lorem ipsum dolor sit amet, consectetur
+                                adipiscing elit, sed do eiusmod tempor
+                                incididunt ut labore et dolore magna aliqua. Ut
+                                enim ad minim veniam, quis nostrud exercitation
+                                ullamco laboris nisi ut aliquip ex ea commodo
+                                consequat. Duis aute irure dolor in
+                                reprehenderit in voluptate velit esse cillum
+                                dolore eu fugiat nulla pariatur. Excepteur sint
+                                occaecat cupidatat non proident, sunt in culpa
+                                qui officia deserunt mollit anim id est laborum.
+                            </Text>
+                        </Flex>
+                        <Flex>
+                            <Card size="5"></Card>
+                            <Card size="2"></Card>
+                            <Card size="2"></Card>
+                        </Flex>
+                    </Flex>
                 </Container>
             </Box>
-            <Box className="bg-gray-200 dark:bg-gray-900" p="7">
-                <Container size="3" className="flex-grow" py="8">
-                    <Heading size="8" align="center" mb="7">
+            <Box p="7" className="flex items-center justify-center">
+                <Card size="3" variant="ghost" className="w-[84%] shadow-lg">
+                    <Heading
+                        size="8"
+                        align="center"
+                        mb="7"
+                        mt="3"
+                        weight="medium"
+                        className="font-mono text-slate-800 dark:text-white"
+                    >
                         Follow us on our socials!
                     </Heading>
                     <Flex justify="center" mb="7">
-                        <Link
+                        <Text className="!mt-px inline-block h-px w-12 border-b border-gray-400 bg-opacity-30 dark:border-orange-400 sm:w-16 md:w-44"></Text>
+                        <Text
                             mx="3"
-                            className="bg-gray-100 p-5 "
+                            className="inline-block h-[4px] w-[4px] rounded-full bg-gray-800 bg-opacity-80 dark:bg-orange-400"
+                        />
+                        <Text className="!mt-px inline-block h-px w-12 border-b border-gray-400 bg-opacity-30 dark:border-orange-400 sm:w-16 md:w-44"></Text>
+                    </Flex>
+                    <Flex justify="center" mb="4">
+                        <Link
+                            mx="2"
+                            className="bg-gray-100"
                             href="https://www.instagram.com/jazzintoronto/ "
                         >
-                            <Button size="4">Instagram</Button>
+                            <Button
```

Comment:
Instead of setting all of these very specific heights and widths on each button, a cleaner way to do this is to limit their width with an outer container that has a fixed width. In general, I would challenge you to use as few specific pixel widths and heights other than the very outer component. In addition, I think there are a bunch of properties we should be setting with radix props. This will help us with theming in the future.